src/dendropy/datamodel/treemodel/_tree.py

```diff
@@ -3096,12 +3096,26 @@ def shuffle_taxa(self, include_internal_nodes=False, rng=None):
         assert len(current_to_shuffled_taxon_map) == len(current_node_taxon_map)
         return current_to_shuffled_taxon_map
 
-    def ladderize(self, ascending=True):
+    def ladderize(self, ascending=True, default_order=False):
         """
-        Sorts child nodes in ascending (if ``ascending`` is |False|) or
-        descending (if ``ascending`` is |False|) order in terms of the number of
-        children each child node has.
+        Sorts child nodes in ascending (if ``ascending`` is |True|) or descending (if
+        ``ascending`` is |False|) order in terms of the number of children each child
+        node has. If ``default_order`` is |True| then leaf nodes get sorted by taxon
+        labels when they are in polytomies.
         """
+
+        if default_order:
+            order = tuple(sorted(leaf.taxon.label for leaf in self.leaf_node_iter()))
+
+        def dorder(node):  # -> int
+            if default_order:
+                try:
+                    return order.index(node.taxon.label)
+                except:
```

Comment:
Hello @davipatti :-)

Using a naked `except` is almost never appropriate (e.g., if the user hits control-C during the `try` the program will not be interrupted). Catch `ValueError` instead.

Reply:
Hello @terrycojones !

No excuses for this one...